src/Raven.Server/Smuggler/Documents/StreamSource.cs

```diff
@@ -1331,28 +1331,38 @@ private async IAsyncEnumerable<DocumentItem> ReadDocumentsAsync(INewDocumentActi
                     var data = builder.CreateReader();
                     builder.Reset();
 
-                    if (data.TryGet(Constants.Documents.Metadata.Key, out BlittableJsonReaderObject metadata) &&
-                        metadata.TryGet(DocumentItem.ExportDocumentType.Key, out string type))
+                    if (data.TryGet(Constants.Documents.Metadata.Key, out BlittableJsonReaderObject metadata))
                     {
-                        if (type != DocumentItem.ExportDocumentType.Attachment)
+                        if (metadata.TryGet(Constants.Documents.Metadata.Collection, out string collectionName))
```

Comment:
this should not be in the StreamSource, rather in core DatabaseSmuggler, so it would work every direction.

Reply:
We have another mechanism for export with filter by collection inside `DatabaseSource`. 
I think we should do it another way on import.